Mathlib/Data/IsROrC/Basic.lean

```diff
@@ -435,9 +435,9 @@ theorem conj_eq_iff_im {z : K} : conj z = z ↔ im z = 0 :=
   (is_real_TFAE z).out 0 3
 #align is_R_or_C.conj_eq_iff_im IsROrC.conj_eq_iff_im
 
-set_option synthInstance.etaExperiment true in
-@[simp]
-theorem star_def : (Star.star : K → K) = conj :=
+-- porting note: @[simp] requires etaExperiment, but the simpNF linter times out regardless
+-- @[simp]
```

Comment:
https://github.com/leanprover-community/mathlib/blob/46b633fd842bef9469441c0209906f6dddd2b4f5/src/analysis/inner_product_space/pi_L2.lean#L560 is the only place in mathlib3 that breaks after removing the simp attribute, I think.